cert/lib/cert/options.rb

```diff
@@ -60,7 +60,17 @@ def self.available_options
                                      env_name: "CERT_KEYCHAIN_PASSWORD",
                                      sensitive: true,
                                      description: "This might be required the first time you access certificates on a new mac. For the login/default keychain this is your account password",
-                                     optional: true)
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :platform,
+                                     env_name: "CERT_PLATFORM",
+                                     description: "Set the provisioning profile's platform (ios, macos)",
+                                     is_string: false,
```

Comment:
any reason why this isn't a string? I see we have to `to_s` this config value everywhere.